### DISCUSSION TITLE
How to Restore App Settings Files from Backup

### DISCUSSION THREAD

**Discussion by VinXmdk** [adirh3/Fluent-Search#851] [Q&A] [answered]
I've been using Fluent Search for a couple months now and really love it, with the vast amount of options and a very pleasing-looking customizable GUI.  Anyway, I had Exported all of my Settings for Backup purposes in case I ever needed to Restore them and have all of the  .json files created as a re4sult.  As it turned out, I had to do a fresh install of Windows 11 today and have reinstalled Fluent Search,  but am unsure how to restore the Settings config files.  It  is possible I have overlooked it, but I can't find an Import option in the app itself.  Do I need to replace the files manually via Explorer?  And if so, can you point me to the file path of the folder, please?  Thanks in advance!  -Matt

**Comment by adirh3** [comment, marked as answer]
Hey, glad to hear you like Fluent Search! 
To import or export settings you need to go to Settings -> Sync -> Import, then select all the .Json files you have in the backup.
Let me know if that doesn't work out for you.

**Reply by VinXmdk** [reply]
Oh, mam... so I did overlook it.  I feel like such an idiot now.  And  I assure you the complete irony of all this is surely not lost on me. :D
Thank you so much!

**Reply by adirh3** [reply]
All good glad it worked out, feel free to share any questions here :)
